runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -49,6 +49,16 @@ default String getIgnoredItems()
 	)
 	void setIgnoredItems(String key);
 
+	@ConfigItem(
+		keyName = "priceType",
+		name = "Price Type",
+		description = "Show High Alchemy prices for the box total (both prices are still available in the tooltip)"
+	)
+	default LootTrackerPriceType priceType()
+	{
+		return LootTrackerPriceType.GRANGE_EXCHANGE;
```

Comment:
grange?

Reply:
Whoops, this is what I get for coding when ill. ty